apps/playground/src/app/modules/components/header-module/header-module.component.html

```diff
@@ -302,9 +315,8 @@ <h2>Play with the Component 😃</h2>
             [appName]="formHeaderComplete.getRawValue().appName"
             [appDescription]="formHeaderComplete.getRawValue().appDescription"
             operatorLogoSrc="assets/illustrations/Logo ANCT.svg"
-            [operatorLogoAlt]="
-              formHeaderComplete.getRawValue().operatorLogoAlt
-            ">
+            [searchBar]="true"
+            [operatorLogoAlt]="formHeaderComplete.getRawValue().operatorLogoAlt">
```

Comment:
Finally, there's a problem with the style : the search bar is not properly aligned with the "Se connecter" button at the right of it and its margin-top is eaten out by 1rem because of the negative margin-bottom before it...